### ISSUE TITLE
Rstudio allows opening the gtn, can we use that for something?

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/training-material#4265]
Maybe it's useful for something? Could we embed useful bits of the GTN or tutorials like tutorial mode, but within R?
![Screenshot 2023-07-10 at 05-48-22 RStudio Server](https://github.com/galaxyproject/training-material/assets/458683/a13db30d-8fa1-44b6-a0a7-678cb427533b)

@nomadscientist can you imagine any crazy cool ideas here?

**Comment by hexylena** [comment]
Like we can open FAQs? 
![Screenshot 2023-07-10 at 05-50-51 RStudio Server](https://github.com/galaxyproject/training-material/assets/458683/8e9ec4af-9412-4109-8578-5f6401ac0e91)

**Comment by nomadscientist** [comment]
Huh - it's tricky because currently the notebooks themselves contain the tutorial
I wonder if this could help solve some of the formatting issues maybe? Not all the fancy Markdown stuff works last I checked in the notebook versions... I'll ponder

**Comment by hexylena** [comment]
yeah It was more for an 'rstudio-first' type approach? I doubt it could help with formatting.
